pyramus/src/main/webapp/templates/students/viewstudent.jsp

```diff
@@ -2831,6 +2833,180 @@
                 <div class="columnClear"></div>
 
               </div>
+              
+              <!--  Student Integration Info Starts -->
+                
+              <div id="integrations.${student.id}" class="tabContent">
+                <div
+                  id="coursesTabRelatedActionsHoverMenuContainer.${student.id}"
+                  class="tabRelatedActionsContainer"></div>
+                  
+                <div class="genericFormSection">
+              	  <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                    <jsp:param name="titleLocale" value="integrations.slice.title"/>
+              	  </jsp:include>
+            	</div>
+            
+                <div class="genericViewInfoWapper ${secureInfoClass}"
+                  id="studentViewIntegrationInfoWrapper">
+
+                  <div class="genericFormSection"
+                    title="${secureInfoTitle}">
+                    <jsp:include
+                      page="/templates/generic/fragments/formtitle.jsp">
+                      <jsp:param name="titleLocale"
+                        value="students.viewStudent.firstNameTitle" />
+                      <jsp:param name="helpLocale"
+                        value="students.viewStudent.firstNameHelp" />
+                    </jsp:include>
+                    <div class="genericViewFormDataText">
+                    ${student.firstName}</div>                       	
+                  </div>
+                  
+                  <div class="genericFormSection"
+                    title="${secureInfoTitle}">
+                    <jsp:include
+                      page="/templates/generic/fragments/formtitle.jsp">
+                      <jsp:param name="titleLocale"
+                        value="students.viewStudent.lastNameTitle" />
+                      <jsp:param name="helpLocale"
+                        value="students.viewStudent.LastNameHelp" />
+                    </jsp:include>
+                    <div class="genericViewFormDataText">${student.lastName}</div>
+                  </div>
+                  
+                  <c:choose>
```

Comment:
When there's only one condition to test, use `<c:if test="">` rather than choose/when. Same with the birthday section below.